### ISSUE TITLE
FNX3-16971 ⁃ [Dark Theme] Low contrast issues with icon colours in collections and open tabs

### ISSUE THREAD

**Issue by AmyYLee** [mozilla-mobile/fenix#3369]
The default icons in collections and open tab favicons in Dark theme fail contrast test (they are using the same colours as light theme). Attached is a colour palette for dark theme versions of the icons. 

![Screen Shot 2019-06-11 at 5 25 09 PM](https://user-images.githubusercontent.com/5992746/59308048-ffa77800-8c6d-11e9-87dd-cc9f4d0a91c5.png)

**Update favicons and collections icons using this colour palette for dark theme**
![](https://user-images.githubusercontent.com/5992746/59307884-798b3180-8c6d-11e9-885c-cf01887251d3.png)

**Comment by ekager** [comment]
And which list should private mode use @AmyYLee ?
